File: examples/H20/robots/sdk_package/example_py/h2x/high_level/stop_move.py
import humanoid_sdk_py.h2x as h2x

h2x_high_level = h2x.H2xHighLevel()

print("=== h2x 停止运动示例 ===")
ret = h2x_high_level.StopMove()
if ret == 0:
    print("停止运动成功")
else:
    print("停止运动失败，错误码：", ret)

File: examples/H20/robots/sdk_package/example/h2x/high_level/stop_move.cpp
#include <iostream>

#include "h2x/h2x_high_level.h"

using namespace humanoid;
using namespace humanoid::h2x;

int main() {
  std::cout << "=== h2x 停止运动示例 ===" << std::endl;
  H2xHighLevel high_level;

  int ret = high_level.StopMove();
  if (ret == 0) {
    std::cout << "成功进入停止运动模式" << std::endl;
  } else {
    std::cout << "进入停止运动模式失败, 错误码：" << ret << std::endl;
    return -1;
  }
  return 0;
}